sdk/identity/identity/test/public/node/clientSecretCredential.spec.ts

```diff
@@ -81,4 +81,19 @@ describe("ClientSecretCredential", function() {
       ]
     })
   );
+
+  it("supports specifying the regional authority", async function() {
+    const credential = new ClientSecretCredential(
+      env.AZURE_TENANT_ID,
+      env.AZURE_CLIENT_ID,
+      env.AZURE_CLIENT_SECRET,
+      {
+        regionalAuthority: RegionalAuthority.AutoDiscoverRegion
```

Comment:
If we're going to have live or recorded test we shouldn't use `AutoDiscoverRegion` as this behavior will vary based on where the test is run.

Reply:
I’ll skip this test on live mode